decidim-core/app/cells/decidim/wadus/show.erb

```diff
@@ -0,0 +1 @@
+WADUS
```

Comment:
What is this ?

Reply:
This comes from a commit to show an example to include new elements in the profile. This commit will be removed in the final version. I've moved to draft the PR to avoid confusion